packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -7861,6 +7861,9 @@ An additional `notes` object can be passed within the `settings` object. The `no
 
 > **Note:** You can't currently manage app provisioning settings via the API. Use the administrator UI.
 
+###### Identity Store Id
+An additional `identityStoreId` can be passed within the `settings` object if application supports identity store. `identityStoreId` should be a valid identity store application id in the same Org.
+
```

Comment:
Suggest to remove this section and put the `identityStoreId` parameter in the **App Settings** parameter table.

Question about the `identityStoreId` value: is it an `appId` of another app created in the same org representing the external identity store app? Or is it a GUID from an external source?

Reply:
it's appId of another app created in the same org.